src/SUMMARY.md

```diff
@@ -1,7 +1,7 @@
 # Summary
 
-[Getting Started](./getting-started.md)
-[About this guide](./about-this-guide.md)
+- [Getting Started](./getting-started.md)
+- [About this guide](./about-this-guide.md)
```

Comment:
I don't think this is right, see [mdBook `SUMMARY.md` prefix syntax](https://rust-lang.github.io/mdBook/format/summary.html#structure)

Reply:
@jieyouxu Yes, you're right!